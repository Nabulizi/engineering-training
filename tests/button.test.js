'use strict';

import Button from '../src/components/button';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Button>Load Data</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});





