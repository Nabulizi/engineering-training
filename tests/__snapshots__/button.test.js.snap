// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  onClick={[Function]}
>
  Load Data
</button>
`;
